.github/workflows/build.yml

```diff
@@ -1,9 +1,8 @@
 name: Builds
 on:
   push:
-    branches: '*'
+    branches: ['master']
```

Comment:
I believe we had `branches: '*'` so that when folks push work to their fork (prior to creating the pull request) they can ensure that the CI is passing prior to PR creation.  Assuming that's the way one accomplishes that particular behavior, what is the "standard workflow"?  Do folks need to first create a PR against the `master`/`main` branch of their fork in order to trigger CI, then, once confirmed, they point at the upstream branch?  I'm just curious and would like to do the right thing here.

Reply:
Yeah fair, that works